api/src/routes/user.test.ts

```diff
@@ -560,6 +564,113 @@ describe('userRoutes', () => {
         expect(testuser).toStrictEqual(publicUser);
       });
     });
+
+    describe('/user/report-user', () => {
+      afterEach(() => {
```

Comment:
```suggestion
      let sendEmailSpy: jest.SpyInstance;
      beforeEach(() => {
        sendEmailSpy =jest
          .spyOn(fastifyTestInstance, 'sendEmail')
          .mockImplementation(jest.fn());
      });
      afterEach(() => {
```
This lets us check the only thing we're interested in: does the route try to send an email.